src/app/app.component.ts

```diff
@@ -158,4 +148,10 @@ export class AppComponent implements OnInit, OnDestroy, AfterViewInit {
     event.preventDefault();
     this.closeTab(tab);
   }
+
+  getStubStrategies(): void {
+    this.httpService
+      .getStubStrategies()
+      .subscribe((response: string[]) => (StubStrategyUtil.reportStubStrategies = response));
+  }
```

Comment:
If new instances of the API call methods are used, please add the new error handling that is being used in #609

Reply:
Added it